site/en/docs/extensions/mv2/content_scripts/index.md

```diff
@@ -152,14 +152,83 @@ They can include JavaScript files, CSS files or both. All auto-run content scrip
 }
 ```
 
-<table class="simple"><tbody><tr><th>Name</th><th>Type</th><th>Description</th></tr><tr id="matches"><td><code>matches</code></td><td>array of strings</td><td><em>Required.</em> Specifies which pages this content script will be injected into. See <a href="/docs/extensions/mv2/match_patterns">Match Patterns</a> for more details on the syntax of these strings and <a href="#matchAndGlob">Match patterns and globs</a> for information on how to exclude URLs.</td></tr><tr id="css"><td><code>css<code></code></code></td><td>array of strings</td><td><em>Optional.</em> The list of CSS files to be injected into matching pages. These are injected in the order they appear in this array, before any DOM is constructed or displayed for the page.</td></tr><tr id="js"><td><code>js<code></code></code></td><td><nobr>array of strings</nobr></td><td><em>Optional.</em> The list of JavaScript files to be injected into matching pages. These are injected in the order they appear in this array.</td></tr><tr id="match_about_blank"><td><code>match_about_blank</code></td><td>boolean</td><td><em>Optional.</em> Whether the script should inject into an <code>about:blank</code> frame where the parent or opener frame matches one of the patterns declared in <code>matches</code>. Defaults to false.</td></tr></tbody></table>
+<table class="simple">
```

Comment:
Sorry for a meta-comment, but, these tables aren't specially interesting. Could it be worth converting them to Markdown proper?

Reply:
There are a couple bits that are a bit novel.

* Cell content spans multiple lines. The only way to do this with our current table support is by having each row in a single line. While I like single line paragraphs in markdown diffs, I find tabular data harder to parse in a single line. Here's a relevant discussion in the [markdown-it repo](https://github.com/markdown-it/markdown-it/issues/406). 
* We currently have IDs assigned to rows. While we can't do this with in markdown, `markdown-it-attrs` does allow us to get close (e.g. ``| `matches` {: #matches } | … | … |``). Perhaps that's sufficient?
* These tables currently have a class `simple` set on them, but it doesn't appear to be used. Good news is it seems `markdown-it-attrs` [allow you to style tables](https://github.com/arve0/markdown-it-attrs#:~:text=32.-,Similar%20for%20tables,-%2C%20attributes).

Here's the before and after for one table. To be honest, I don't see either approach as a clear win. I'm happy to go with whatever you prefer, @samthor.

```
<table class="simple">
  <tbody>
    <tr>
      <th>Name</th>
      <th>Type</th>
      <th>Description</th>
    </tr>
    <tr id="matches">
      <td><code>matches</code></td>
      <td>array of strings</td>
      <td><em>Required.</em> Specifies which pages this content script will be injected into. See <a
          href="/docs/extensions/mv2/match_patterns">Match Patterns</a> for more details on the
        syntax of these strings and <a href="#matchAndGlob">Match patterns and globs</a> for
        information on how to exclude URLs.</td>
    </tr>
    <tr id="css">
      <td><code>css</code></td>
      <td>array of strings</td>
      <td><em>Optional.</em> The list of CSS files to be injected into matching pages. These are
        injected in the order they appear in this array, before any DOM is constructed or displayed
        for the page.</td>
    </tr>
    <tr id="js">
      <td><code>js</code></td>
      <td>
        <nobr>array of strings</nobr>
      </td>
      <td><em>Optional.</em> The list of JavaScript files to be injected into matching pages. These
        are injected in the order they appear in this array.</td>
    </tr>
    <tr id="match_about_blank">
      <td><code>match_about_blank</code></td>
      <td>boolean</td>
      <td><em>Optional.</em> Whether the script should inject into an <code>about:blank</code> frame
        where the parent or opener frame matches one of the patterns declared in
        <code>matches</code>. Defaults to false.</td>
    </tr>
  </tbody>
</table>
```

```
| Name | Type | Description |
| --- | --- | --- |
| `matches` {: #matches } | array of strings | _Required._ Specifies which pages this content script will be injected into. See [Match Patterns](/docs/extensions/mv2/match_patterns) for more details on the  syntax of these strings and [Match patterns and globs](#matchAndGlob) for information on how to exclude URLs. |
| `css` {: #css } | <nobr>array of strings</nobr> | _Optional._ The list of CSS files to be injected into matching pages. These are injected in the order they appear in this array, before any DOM is constructed or displayed for the page. |
| `js` {: #js } | <nobr>array of strings</nobr> | _Optional._ The list of JavaScript files to be injected into matching pages. These are injected in the order they appear in this array. |
| `match_about_blank` {: #match_about_blank } | boolean | _Optional._ Whether the script should inject into an `about:blank` frame where the parent or opener frame matches one of the patterns declared in `matches`. Defaults to false. |
```

Reply:
Yeah, thanks for the work. I think Markdown is probably better here.